app/assets/javascripts/govuk_publishing_components/components/govspeak.js

```diff
@@ -2,25 +2,27 @@ window.GOVUK = window.GOVUK || {}
 window.GOVUK.Modules = window.GOVUK.Modules || {};
 
 (function (Modules) {
-  'use strict'
+  function Govspeak () { }
 
-  Modules.Govspeak = function () {
-    this.start = function ($element) {
-      if (!$element.hasClass('disable-youtube')) {
-        this.embedYoutube($element)
-      }
+  Govspeak.prototype.start = function ($module) {
+    this.$module = $module[0]
 
-      this.createBarcharts($element)
+    if (!this.$module.classList.contains('disable-youtube')) {
```

Comment:
I'd suggest using `indexOf` instead of `classList` here - this will have the same end result, no increase in lines of code, slightly less legibility - and gives increased (old) Internet Explorer support:

```suggestion
    if (this.$module.className.indexOf('disable-youtube') === -1) {
```